recipe/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: ipylab
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/i/ipylab/ipylab-{{ version }}.tar.gz
+  sha256: c4f0744b1f96d6c4605b986aa59ebccd64451bf71c9483a019ab7a52c5656170
+
+build:
+  number: 0
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation --disable-pip-version-check
+
+requirements:
+  host:
+    - hatch-jupyter-builder >=0.8.2
```

```suggestion
    - hatch-jupyter-builder >=0.5
```

https://github.com/jtpio/ipylab/blob/v1.0.0/pyproject.toml#L47